Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -723,21 +723,16 @@ class ByteBufferTest: XCTestCase {
         XCTAssertEqual(otherBuf, buf)
 
         // Write to both buffers.
-        let firstResult = buf!.withUnsafeMutableWritableBytes { (ptr: UnsafeMutableRawBufferPointer) -> Bool in
```

Comment:
actually, sorry, this needs to be undone. This test is specifically testing `withUnsafeMutableWritableBytes`